rpm/generic/zfs-kmod.spec.in

```diff
@@ -52,6 +52,8 @@ URL:            http://zfsonlinux.org/
 Source0:        %{module}-%{version}.tar.gz
 Source10:       kmodtool
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id} -u -n)
+BuildRequires:  gcc, make
+BuildRequires:  elfutils-libelf-devel
```

Comment:
You'll want to wrap these `BuildRequires` in a conditional, or otherwise it'll fail package builds for Debian.

So, this should be:
```rpm-spec
%if 0%{?rhel}%{?fedora}
BuildRequires:  gcc, make
BuildRequires:  elfutils-libelf-devel
%endif
```